daemon/daemon.go

```diff
@@ -575,6 +581,12 @@ func (daemon *Daemon) RestartSwarmContainers() {
 	// RLIMIT_NOFILE is small to avoid EMFILE).
 	parallelLimit := adjustParallelLimit(len(daemon.List()), 128*runtime.NumCPU())
 
+	// Windows containers are much larger than Linux containers and each of them
+	// have > 20 system processes which why it is better to restore them without parallelism.
+	if runtime.GOOS == "windows" {
```

Comment:
I notice that `adjustParallelLimit` is already platform specific; instead of adjusting it here, should we perhaps make that function always return `1` on Windows? https://github.com/moby/moby/blob/ffabf0d542f16b9fc6a69f096312656dd190d590/daemon/daemon_windows.go#L42-L45

Reply:
Very good point. I totally forgot to check what adjustParallelLimit function actually does. Changed.